### PR TITLE
test: Improve tests of base_order_utils

### PR DESCRIPTION
**Pull request by tevrat-aksoy**
# Pull Request type

Please add the labels corresponding to the type of changes your PR introduces:

- Testing

## What is the current behavior?

Resolves: #484

### CODE REVIEW

File: tests/order/test_base_order_utils.cairo [hunk 1]
```diff
@@ -1,51 +1,229 @@
...
+    );
+
+    assert(true, 'e');
+}
+
+#[test]
+#[should_panic(
+    expected: ('invalid_order_price', 100000, 200000, 199999, 6053968548023263173723725853541)
+)]
+fn given_limit_increase_price_higher_than_trigger_when_validate_order_trigger_price_then_works() {
```
Comment (zarboq) [inline tests/order/test_base_order_utils.cairo:153]: Some functions are named `...then_works` but it should be `...then_panic`

**Review by zarboq — CHANGES_REQUESTED** [review]
Great, only some small naming issues on panic test

File: tests/order/test_base_order_utils.cairo (hunk 1 above)
Comment (tevrat-aksoy) [reply tests/order/test_base_order_utils.cairo:153]: sorry, I forgot about that. I will fix it

Comment (tevrat-aksoy) [reply tests/order/test_base_order_utils.cairo]: fixed here
https://github.com/tevrat-aksoy/satoru/commit/b592da250dee903e5ccbfea38383ccebf9e98dd3

**Review by zarboq — APPROVED** [review]

**Pull request merged**